modules/geospatial/src/ellipsoid/ellipsoid.js

```diff
@@ -98,6 +98,9 @@ export default class Ellipsoid {
   // Converts the provided cartesian to cartographic (lng/lat/z) representation.
   // The cartesian is undefined at the center of the ellipsoid.
   cartesianToCartographic(cartesian, result = [0, 0, 0]) {
+    if (Array.isArray(cartesian)) {
```

Comment:
There is a `Vector3.from` method that converts the input from array/object etc.

You could use something like:

`cartesian = scratchCartesian.from(cartesian)`